src/multilspy/language_server.py

```diff
@@ -1539,43 +1559,72 @@ def load_cache(self):
                 # cache often becomes corrupt, so just skip loading it
                 self.logger.log(
                         f"Failed to load document symbols cache from {self._cache_path}: {e}. Possible cause: the cache file is corrupted. " 
-                        "Check for any errors related to saving the cache in the logs.", 
+                        "Check for any errors related to saving the cache in the logs.",
                         logging.ERROR
                     )
 
 
+    async def request_workspace_symbol(self, query: str) -> Union[List[multilspy_types.UnifiedSymbolInformation], None]:
```

Comment:
Great! We should adjust our find_symbol to use this instead of our workaround

Reply:
Added #44